uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/Relationship.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class CRM_HRCore_Test_Fabricator_Relationship {
+
+  private static $defaultParams = [
+    'sequential' => 1,
+    'name_a_b' => "test AB",
+    'name_b_a' => "test BA",
+    'contact_type_a' => "Individual",
+    'contact_type_b' => "Individual",
+  ];
+
+  public static function fabricateRelationshipType($params = []) {
+    $params = array_merge(self::$defaultParams, $params);
+
+    $result = civicrm_api3(
+      'Activity',
```

Comment:
nope , it is just a tests that I forget to run since I was running them one by one ... thanks for your note about installing all extensions in the base class headless method I now can just run all the tests at once so no chance to forget anything.